.gitignore

```diff
@@ -29,6 +32,7 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
+Pipfile.lock
```

Comment:
Pipfile.lock is supposed to be part of the source code

Reply:
Really? I've thought that it is auto-generated by `pipenv install`.

Reply:
There is Pipfile.lock in .gitingore in our testsuite too, see https://github.com/3scale-qe/3scale-tests/blob/main/.gitignore#L42